src/ui/components/search/searchcomponent.js

```diff
@@ -128,10 +128,21 @@ export default class SearchComponent extends Component {
      *
      * If no redirectUrl provided, we keep the page as a single page app.
      *
-     * @type {boolean}
+     * @type {string}
      */
     this.redirectUrl = config.redirectUrl || null;
 
+    /**
+     * redirectUrlTarget will force the search query submission to open in the frame specified if
+     * redirectUrl is also supplied.
+     * Optional, defaults to current frame.
+     *
+     * If no redirectUrlTarget provided, we keep the page as a single page app.
```

Comment:
Minor nit, but having this sentence and "Optional, defaults to current frame" is a bit redundant.

Reply:
Oh hahah totally makes sense, updating! this is what I get for copy-pasting the JSDoc from the redirectUrl 🙈